frontend/src/components/cluster-dashboard/cluster-graph-view/ClusterGraphView.scss

```diff
@@ -0,0 +1,174 @@
+@import "./../../../assets/styles/variables.scss";
+@import "./../../../assets/styles/common.scss";
+
+.shukra-right-container {
+  & .shukra-visual-title {
+    text-transform: uppercase;
+    color: $primary-color;
+    height: 40px;
+    padding: $single-spacing;
+    font-size: $text-size-lg;
+    font-weight: bold;
+    text-align: center;
+  }
+
+  & .shukra-visual-wrapper {
+    height: calc(100vh - 140px);
+    position: relative;
+
+    & .vis-network {
+      outline: none;
+
+      & div.vis-navigation div.vis-button.vis-up,
+      div.vis-navigation div.vis-button.vis-down,
+      div.vis-navigation div.vis-button.vis-left,
+      div.vis-navigation div.vis-button.vis-right,
+      div.vis-navigation div.vis-button.vis-zoomIn,
+      div.vis-navigation div.vis-button.vis-zoomOut,
+      div.vis-navigation div.vis-button.vis-zoomExtends {
+        background-image: none !important;
+        padding: 0px $single-spacing;
+      }
+
+      & div.vis-navigation div.vis-button:hover {
+        box-shadow: none !important;
+      }
```

Comment:
is scoping this css to just divs intended or necessary? If not, I would nix it. (but I know how CSS can force your hand in some cases)

Reply:
Nah, and on second thoughts, I guess I will remove the divs.